cmd/godns/godns.go

```diff
@@ -71,9 +71,9 @@ func dnsLoop() {
 	for _, domain := range configuration.Domains {
 		domain := domain
 		if configuration.RunOnce {
-			ddnsHandler.DomainLoop(&domain, panicChan, configuration.RunOnce)
+			ddnsHandler.UpdateIP(&domain)
```

Comment:
Being able to call the single run function directly here is what makes removing the `runOnce` logic from handler possible.